src/views/SecondaryBarView.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { useMemo } from 'react';
+
+import {
+	Accordion,
+	type AccordionItemType,
+	Avatar,
+	Container,
+	IconButton,
+	Padding,
+	Text,
+	Tooltip
+} from '@zextras/carbonio-design-system';
+import { useUserAccount } from '@zextras/carbonio-shell-ui';
+import { noop } from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { ROUTES_INTERNAL_PARAMS } from '../constants';
+import { useNavigation } from '../hooks/useNavigation';
+
+const StyledText = styled(Text)`
+	min-width: 0;
+	flex-basis: 0;
+	flex-grow: 1;
+`;
+
+const AccordionItem = ({ item }: { item: AccordionItemType }): React.JSX.Element => (
+	<Container
+		orientation="horizontal"
+		mainAlignment="flex-start"
+		padding={{ all: 'small' }}
+		height="2.5rem"
+		style={{ minWidth: 0, flexBasis: 0, flexGrow: 1 }}
+	>
+		<Padding right="small">
+			<Avatar label={item.label ?? ''} />
+		</Padding>
+		{item.label && (
+			<StyledText size="medium" {...item.textProps}>
+				{item.label}
+			</StyledText>
+		)}
+	</Container>
+);
+
+const SecondaryBarView = ({ expanded }: { expanded: boolean }): React.JSX.Element => {
+	const [t] = useTranslation();
+	const { name } = useUserAccount();
+	const { navigateTo } = useNavigation();
+	const { pathname } = useLocation();
+
+	const items = useMemo(
+		(): AccordionItemType[] => [
+			{
+				id: 'id1',
+				label: name,
+				CustomComponent: AccordionItem,
+				open: true,
+				items: [
+					{
+						id: 'id2',
+						icon: 'PeopleOutline',
+						label: t('secondaryBar.myContactGroups', 'My Contact Groups'),
+						onClick: (ev): void => {
+							ev.stopPropagation();
+							navigateTo(ROUTES_INTERNAL_PARAMS.route.contactGroups);
+						},
+						active: pathname.endsWith(`${ROUTES_INTERNAL_PARAMS.route.contactGroups}`)
```

Comment:
We need to replace the `endsWith` with an `includes`, because otherwise the entry is not active when the displayer is open. I noticed this problem in our branch, you might want to fix it also here.